src/server.py

```diff
@@ -78,6 +81,37 @@ def get_date_range(dates):
                 'endDate': max(dates).strftime('%Y-%m-%d %H:%M')}
 
 
+def valve(head_rev, pushlog_id):
+    """Determine which kind of job should been returned"""
+    low_job_return = True
+    COUNTER.increase_the_counter()
+    request_list = []
+    try:
+        request_list = session.query(TaskRequests.head_rev, TaskRequests.pushlog_id,
+                                     TaskRequests.low_job_return).all()
```

Comment:
I'm not sure I understand the 'table will hold' in here. Here is the database table:

> id |                 head_rev                 | pushlog_id | low_job_return 
> ----+------------------------------------------+------------+----------------
>  1 | 0bd27824832d4213a90ec0ef08de422b33589358 | 80484      | t

And it will return a list of set like **[(0bd27824832d4213a90ec0ef08de422b33589358, 80484, True)]**. Does it help?